src/tox_current_env/hooks4.py

```diff
@@ -209,6 +209,12 @@ def prepend_env_var_path(self):
     def runs_on_platform(self):
         return sys.platform
 
+    @classmethod
+    def python_spec_for_path(cls, path):
+        # https://github.com/fedora-python/tox-current-env/issues/77
+        ver, is_64 = sys.version_info, sys.maxsize != 2**32
```

Comment:
`sys.maxsize` is `2**31-1` for 32 bit and `2**63-1` for 64bit so the condition should be:

```suggestion
        ver, is_64 = sys.version_info, sys.maxsize > 2**32
```